### PR TITLE
Add another implementation to plusOne.py

### PR DESCRIPTION
**Pull request by Mohamed3on**
Solve the problem by converting the list to a string, then to an integer, then incrementing that integer and returning it as a list.

### CODE REVIEW

File: array/plus_one.py [hunk 1]
```diff
@@ -36,3 +36,13 @@ def plus_one(digits):
     new_num = [0] * (n+1)
     new_num[0] = 1
     return new_num
+## another implementation, convert list to string, then to integer, then increment that integer
+def plusone(digits):
+        num=""
+        for d in digits:
+            num+=str(d)
+        incremented=str(int(num)+1)
+        incrementedAsList=[]
```
Comment (venky18) [inline array/plus_one.py]: you can convert  `string` to `list`  by  using `list`
so you can replace the last three lines as `incrementedAsList = list(incremented) `

Comment (Mohamed3on) [inline array/plus_one.py]: Indeed, but it would convert it to a list of chars, while the needed return type was a list of ints, that's why I used a loop.

Comment (Mohamed3on) [inline array/plus_one.py]: For reference:
https://leetcode.com/problems/plus-one/#/description

**Comment by Mohamed3on** [comment]
Is there something wrong with the implementation? Or do you not want to add another implementation?

File: array/plus_one.py (hunk 1 above)
Comment (keon) [inline array/plus_one.py]: could you convert this to snake_case ?

**Comment by ofek** [comment]
This solution is sub-optimal, but if accepted here is the condensed version:

```python
def plus_one(digits):
    return [int(s) for s in str(int(''.join(str(i) for i in digits)) + 1)]
```

**Comment by ankit167** [comment]
@Mohamed3on Closing this request, as it has conflicts with master. Please push a fresh request.

**Pull request closed without merging**